src/server/app/admin/controllers/users/users.authentication.server.controller.js

```diff
@@ -73,12 +75,58 @@ function adminCreateUser(user, req, res) {
 }
 
 
+// Send email alert about new account request
+function signupAlert(user, req, res) {
+	let defer = q.defer();
+
+	res.render('../../admin/templates/user-signup-alert-email', {
+		name: user.name,
+		username: user.username,
+		appName: config.app.instanceName,
+		url: 'http://' + req.headers.host + '/#/admin/users'
```

Comment:
So I just commited a change that adds the url to the config and pulls that instead of pulling from the header.  Let me know what you guys think.